Exercise_B/index.html

```diff
@@ -9,8 +9,9 @@
     <link rel="stylesheet" href="index.css" />
   </head>
   <body>
-    <!-- CODE HERE -->
-
+    <div class="box1"></div>
+    <div class="box2"></div>
+    <div class="box3"></div>
     <script src="" async defer></script>
   </body>
 </html>
```

Comment:
Hey Anson,
Nice work on this exercise, everything looks in order and you have made good use of Flex